molo/surveys/templates/wagtailsurveys/list_forms.html

```diff
@@ -1,24 +0,0 @@
-{% load i18n molo_survey_tags %}
-<table class="listing">
-    <col width="50%"/>
-    <col width="50%"/>
-    <thead>
-        <tr>
-            <th class="title">{% trans "Title" %}</th>
-            <th class="type">{% trans "Origin" %}</th>
-        </tr>
-    </thead>
-    <tbody>
-        {% get_survey_list_for_site as survey_pages %}
```

Comment:
Do we still need this template tag? A search of the code suggests these two files are the only places it is used?

Reply:
Good catch! I've also check `molo-gem` and `nurseconnect` and they don't use it either. I'll remove it.